.github/workflows/ci.yml

```
@@ -0,0 +1,54 @@
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
```

actionlint (1.7.11)

[error] expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)